README.md

```diff
@@ -1,300 +1,12 @@
-| plot_parcel_history.py        | Create a scatter plot with the original and final parcel height. |
-
-These scripts allow you to explore Eulerian and Lagrangian simulation data.
-Information how to use these scripts is printed when running with the flag `--help`, e.g. `plot_cross_sections.py --help`.
-
-> [!TIP]
-> We suggest that you append the plotting directory to the `$PATH` environment variable with
-> ```bash
-> export PATH=$PATH:/work/d185/d185/$USER/workshop-trieste/plotting
-> ```
-> in order to facilitate the usage of the plotting scripts. After this operation you can, for example, simply
-> type `plot_cross_sections.py --help` from any directory and without prefixing the Python interpreter.
-
-For example, when you run the moist bubble case with the default settings, you can create a parcel cross section plot with
-```bash
-plot_intersected_ellipses.py --filename moist_0000000005_parcels.nc \
-                              --steps 0 1 2 3 4 5 \
-                              --dataset buoyancy \
-                              --plane xz
-```
-which gives you this plot.
-
-![Cross sections showing the ellipses obained from the intersection of the ellipsoids with the xz-plane through the centre of the y-axis.](figures/xz-intersected_ellipses_location_32_buoyancy.png
-"Cross sections showing the ellipses obained from the intersection of the ellipsoids with the xz-plane through the centre of the y-axis.")
-
-### Plotting with xarray
-Alternatively, you can use the package [xarray](https://docs.xarray.dev/en/stable/user-guide/io.html) to visualize the data.
-> [!NOTE]
-> When loading the `pandas` package two warnings are thrown regarding the version of the packages `numexpr` and `bottleneck`. To suppress these warnings, you can use
-> ```Python
-> import warnings
-> warnings.filterwarnings("ignore", module='pandas')
-> ```
-The following code snippet creates a cross section plot of the gridded buoyancy field.
-```Python
-import warnings
-warnings.filterwarnings("ignore", module='pandas')
-
-import matplotlib.pyplot as plt
-import colorcet as cc
-import xarray as xr
-
-
-da = xr.open_dataset("moist_fields.nc")
-
-buoy = da.buoyancy
-
-# get a time series of xz-cross sections through the centre of the y-axis (grid point 32)
-t_buoy2d = buoy.isel(t=slice(0, 24, 4), y=32)
-
-g = t_buoy2d.plot(x="x", y="z", col="t", col_wrap=3, cmap=cc.cm['coolwarm'], figsize=(12, 7))
-
-# fix the time format
-times = da.t.dt.strftime("%H:%M:%S")
-for i, ax in enumerate(g.axs.flat):
-    ax.set_title("Time "+ times.data[i] + " (HH:MM:SS)")
-
-plt.savefig('xz-buoyancy-cross_section.png', bbox_inches='tight', dpi=200)
-plt.close()
-```
-
-![Cross sections through the centre of the y-axis.](figures/xz-buoyancy-cross_section.png
-"Cross sections through the centre of the y-axis.")
+- [Accessing the Cirrus cluster](sites/cluster_access.md)
+- [Setting up the environment](sites/setup_environment.md)
+- [Moist bubble test case](sites/moist_bubble.md)
```

Comment:
I would maybe number the files so they show up "in order", like:
```
01-cluster_access.md
02-setup_environment.md
03-moist_bubble.md
```

Reply:
OK. Done.